File: test/ocamlTests/CharTest/toLowercaseTest.ml
test "toLowercase('A')" (fun () -> expect (Char.toLowercase 'A') |> toEqual Eq.char 'a') ; 
test "toLowercase('a')" (fun () -> expect (Char.toLowercase 'a') |> toEqual Eq.char 'a') ; 
test "toLowercase('7')" (fun () -> expect (Char.toLowercase '7') |> toEqual Eq.char '7') ; 
test "toLowercase('\233')" (fun () -> expect (Char.toLowercase '\233') |> toEqual Eq.char '\233') ; 

File: test/fsharpTests/CharTest/toLowercaseTest.fs
testCase "toLowercase('A')" 
<| fun _ -> 
    let expected = 'a'
    Expect.equal expected (Char.toLowercase 'A') "error"
testCase "toLowercase('a')" 
<| fun _ -> 
    let expected = 'a'
    Expect.equal expected (Char.toLowercase 'a') "error"
testCase "toLowercase('7')" 
<| fun _ -> 
    let expected = '7'
    Expect.equal expected (Char.toLowercase '7') "error"
testCase "toLowercase('\233')" 
<| fun _ -> 
    let expected = '\233'
    Expect.equal expected (Char.toLowercase '\233') "error"
